manifests/installer_telemetry.pp

```diff
@@ -0,0 +1,54 @@
+# This class handles the installation telemetry for the Datadog installer.
+#
+# @param api_key String:Your DataDog API Key. Please replace with your key value.
```

Comment:
❓ Question :
Is there a need to ask to replace with the customer API Key since this variable get's overwritten everytime it gets called ?

Reply:
Nope indeed, I'll remove the "Please replace with your key value"